app/assets/stylesheets/custom.scss

```
@@ -124,3 +125,15 @@ input {
   margin-left: 0;
   width: auto;
 }
+
+/* Users index */
+.users {
+  list-style: none;
+  margin: 0;
+
+  li {
+    overflow: auto;
+    padding: 10px 0;
+    border-bottom: 1px solid light-gray;
```

```suggestion
    border-bottom: 1px solid light-gray;
```
Đặt như này thì làm sao chạy được nhỉ